src/libstd/rt/unwind.rs

```diff
@@ -160,7 +160,7 @@ pub fn panicking() -> bool {
 // An uninlined, unmangled function upon which to slap yer breakpoints
 #[inline(never)]
 #[no_mangle]
-fn rust_panic(cause: Box<Any + Send>) -> ! {
+pub fn rust_panic(cause: Box<Any + Send>) -> ! {
```

Comment:
The symbol is not available to link against but binaries typically still retain metadata about other symbols (even private ones) and in this case the metadata for `rust_panic` is kept around.